source/docs/software/advanced-controls/controllers/bang-bang.rst

```diff
@@ -0,0 +1,65 @@
+Bang-Bang Control with BangBangController
+=========================================
+
+.. warning:: Bang-bang control is an extremely aggressive algorithm that relies on response asymmetry to remain stable.  Be *absolutely certain* that your motor controllers have been set to "coast mode" before attempting to control them with a bang-bang controller, or else the braking action will fight the controller and cause potentially destructive oscillation.
+
+`"Bang-bang" control <https://en.wikipedia.org/wiki/Bang%E2%80%93bang_control>`__ is a control strategy that employs only two states: on and off.  This is roughly equivalent to a P loop with infinite gain.
```

Comment:
I would add a sentance that says it's on when it's below the setpoint and off when it's above the setpoint.